databases/week1/sql_queries.sql

```diff
@@ -0,0 +1,89 @@
+--Find out how many tasks are in the task table
+SELECT
+  count(id)
+FROM
+  task;
+
+--Find out how many tasks in the task table do not have a valid due date
+SELECT
+  count(id)
+FROM
+  task
+WHERE
+  due_date IS NULL;
+
+--Find all the tasks that are marked as done
+SELECT
+  *
+FROM
+  task
+  JOIN status ON task.status_id = status.id
+WHERE
+  status.name LIKE 'DONE';
+
+--Find all the tasks that are not marked as done
+SELECT
+  *
+FROM
+  task
+  JOIN status ON task.status_id = status.id
+WHERE
+  status.name NOT LIKE 'DONE';
+
+--Get all the tasks, sorted with the most recently created first
+SELECT
+  *
+FROM
+  task
+ORDER BY
+  created DESC;
+
+--Get the single most recently created task
+SELECT
+  *
```

Comment:
Same as above :)

Reply:
Done with only two columns of task title and created column